docs/model-serving/quickstart.md

```diff
@@ -0,0 +1,175 @@
+
+By default `connection.get_model_serving()` returns the Model Serving of the project you are working with.
+
+=== "Python"
+
+    ```python
+    import hsml
+
+    # Create a connection
+    connection = hsml.connection()
+
+    # Get the model serving handle for the project's model serving
+    ms = connection.get_model_serving()
+    ```
+
+
+### Integration with the Model Registry
+
+The Model Serving library integrates smoothly with the Model Registry library. Metadata objects such as [Model](../generated/model-registry/model.md) can be used in multiple methods in the Model Serving library to simplify the code and avoid duplication of parameters.
+
+As a result, the easiest way to deploy an existing model is using the `model.deploy()` method available in the model metadata object. For instance, the version with highest accuracy of a trained model can be deployed with the following lines.
+
+=== "Python"
+
+    ```python
+    # Get the model registry handle for the project's model registry
+    mr = connection.get_model_registry()
+
+    # Get the version with highest accuracy
+    model = mr.get_best_model('mnist', 'accuracy', 'max')
+
+    # Create a deployment
+    deployment = model.deploy()
+    ```
+
+To learn more about managing models with the Model Registry see the [Model Registry documentation](../model-registry/quickstart.md).
+
+
+### Deployment
+
+Assuming you have already created a model in the Model Registry, a deployment can now be created to prepare a model artifact for this model and make it accessible for running predictions behind a REST endpoint. A model artifact is a package containing all necessary files for the deployment of a model, including the model file and/or custom scripts for loading the model (a.k.a. predictor script) or transforming the model inputs at inference time (a.k.a. transformer script).
+
+See the [example notebooks](https://github.com/logicalclocks/hops-examples/blob/master/notebooks/ml).
+
+#### Predictors and Transformers
+
+Predictors are in charge of running a model server that loads a trained model, handles inference requests and returns predictions (see the [Predictor Guide](../generated/model-serving/predictor.md)). These inference requests can be transformed at inference time by configuring a transformer.
+
+Transformers are used to apply transformations on the model inputs before sending them to the model for making predictions (see the [Transformer Guide](../generated/model-serving/transformer.md)).
+
+As an example, consider a model trained to detect unusual activity of users within a platform. Instead of sending all the recent activity records of a user each time we want to predict if the use is abnormal, we can simply send the _user_id_ and build/enrich the model input in the transformer before sending it to the predictor, by retrieving the most recent activity from an external source such as a [Feature Store](https://docs.hopsworks.ai/feature-store-api/latest/).
```

Comment:
, by retrieving the most recent activity
->
. We enrich the model input by retrieving the most recent user activity from the [Feature Store] using the _user_id_ as an entity key